.gitlab/ci/.gitlab-ci.global.yml

```diff
@@ -468,6 +468,7 @@
           fi
         fi
       fi
+    - cp -r coverage_report artifacts/coverage_report || true
```

Comment:
i think is better not to use the `|| true` as if there is any error we will see it here 
instead to fail on the `upload_code_coverage_report ` without details